certified-connectors/Jupyrest/apiDefinition.swagger.json

```diff
@@ -0,0 +1,294 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "x-ms-api-annotation": {
+      "status": "Production"
```

Comment:
@litiany23 please update it to `"Preview"` as all the new connectors should be deployed with the Preview tag.  In order to update it to the "Production" tag, we require your connector to meet the following criteria after publishing it to all the production regions:

1. Connector has 99.95% reliability (calculated by % of 5xx (-502 & -504) errors) 
2. Connector has 99.5% SLO (percentage of connections meeting SLA bar above)
3. Connector has greater than 80% success rate (Calculated by % of 4xx and 5xxx errors) 
4. Connector has at least 500 weekly active users over the last 3 weeks.
5. Backend service APIs, that the connector is calling, needs to be in PRODUCTION.
6. Partner should be able provide 24/7 support for high severity incidents.

Reply:
Updated it to Preview. Thanks